_positions/product-lead.md

```diff
@@ -8,6 +8,9 @@ active: true
 ###Role Summary
 An 18F Product Lead is responsible for managing the development, delivery, and continuous improvement of products while establishing and managing our relationship with our partners (clients). As a product lead, you’ll work with the 18F intake team to evaluate potential work and communicate to potential clients, partner with the finance and contracts team to close deals, and work with cross-functional teams to deliver the right product in the right way.
 
+At this time, the Product team is only looking for new members who are able to join the team at an office - Chicago, NYC, DC, San Francisco. 
```

Comment:
A few questions and comments:
* Please change the hyphen to an em-dash (on a Mac, you can type that by pressing `shift-option-hyphen`).
* Let's change `DC` to `D.C.`, per the style guide.
* Do we still technically have a NYC office? If so, let's update https://pages.18f.gov/joining-18f/who-we-are-hiring/ accordingly (in a separate pull request).
* We could parallel the "work out of our offices" phrasing from that page, something like "At this time, we're only hiring Product Leads who are able to work out of our offices in [city list] or [final city]." How does that sound to you?